orchestrator/relayer/batch_relaying.go

```diff
@@ -161,16 +161,27 @@ func (s *peggyRelayer) RelayBatches(
 				continue
 			}
 
+			// Checking in pending txs(mempool) if tx with same input is already submitted
+			// We have to check this at the last moment because any other relayer could have submitted
```

Comment:
This is a no-op if the alchemy API is not provided, right?

```suggestion
			// We have to check this at the last moment because any other relayer could have submitted.
```

Reply:
Yes, it will try to iterate through the pending tx list, but since nothing is there it will just return false.